### ISSUE TITLE
The "index" api link is not an index. 

### ISSUE THREAD

**Issue by Pomax** [nodejs/nodejs.org#5333]
### URL:

https://nodejs.org/api/index.html

### Browser Name:

n/a

### Browser Version:

n/a

### Operating System:

n/a

### How to reproduce the issue:

1. Visit https://nodejs.org/api/index.html
2. note that this is identical to the nav system on the left

That's it, that's the whole bug, the index page is _not_ an documentation index page in the slightest. This should be a very long page with all terms across all documentation so that people can search and then click through to the thing they're looking for.

This is an index: https://docs.oracle.com/en/java/javase/18/docs/api/index-files/index-1.html

The nodejs page is not.

**Comment by ovflowd** [comment]
What you probably want to achieve is https://nodejs.org/api/all.html no?

**Comment by ovflowd** [comment]
Also, this is not really a bug, so I've removed the bug label :)

**Comment by ljharb** [comment]
I'm confused, there's nothing about "index.html" that requires it to actually BE an index - that's just the history of where the name comes from.

**Comment by ovflowd** [comment]
I feel like the `/index.html` in the past was supposed to land to https://nodejs.org/api/documentation.html

Something might have changed, as on https://nodejs.dev/en/api we show the "documentation.html"

**Comment by ovflowd** [comment]
@richardlau did we accidentally change something on the API side of things with NGINX? (I don't recall such change, but just double-checking)

**Comment by richardlau** [comment]
Not that I'm aware of. You can check with https://web.archive.org/ what was being returned.

**Comment by Pomax** [comment]
index.html is the _file_ that for the longest time most things automatically redirect to (but tons of things don't, it's still a manual thing you need to configure in plenty of servers), but the location, when used in online documentation like API docs, is "top" or "home", not "index". That word, _in the context of online documentation_ still only has the very specific meaning of being the index list of terms used in that documentation.

**Comment by ljharb** [comment]
API documentation is indeed a regular website. The only context where anyone should expect "index.html" to be an index is a filesystem directory listing.

**Comment by ovflowd** [comment]
I agree with @ljharb here, but I appreciate the healthy discussion about what the API docs index should or should not be (and what an index file even is).

I feel we might be derailing here. 

Also, @richardlau seems I was wrong (https://web.archive.org/web/20190122151446/https://nodejs.org/api/index.html) index was just an "index".

I'm all-in onwards on changing this behaviour as I also would prefer the main "documentation.html" on serving as the "index".

The question now is, if other collaborators are happy with this. Also, this issue should be moved to `nodejs/node` as it is not (currently) on the realm of the Website Team regarding any changes on the living API docs.

If anybody from @nodejs/collaborators could transfer this issue to `node` that'd be great!

**Comment by cclauss** [comment]
cclauss: What is index.htlm?

ChatGPT: index.html is the default file name for the home page or the starting point of a website. It is a standard web page that is displayed when someone visits a website's main URL. 

When a user types a website's domain name into their web browser's address bar, the web server will automatically look for a file named index.html in the website's root directory and display its content on the user's screen. If an index.html file is not present, the server may display a directory listing or a 404 error page instead.

The content of the index.html file typically contains HTML, CSS, and JavaScript code that defines the structure, layout, and interactive elements of the website's home page. It may also include links to other pages on the website, images, videos, and other media.

**Comment by ovflowd** [comment]
I'd argue this is now a stale issue, and I doubt there's any short-term actionable here as the Docs are generated the way they are. Not to mention these docs are not generated on the Website repository. So I'd argue this should be handled by core folks.

I will move this to `nodejs/core` and see what the triggers and collaborators there think about this.
